src/Workspaces/Core/Portable/Workspace/Host/PersistentStorage/IPersistentStorageLocationService.cs

```diff
@@ -21,6 +23,13 @@ internal interface IPersistentStorageLocationService : IWorkspaceService
     [ExportWorkspaceService(typeof(IPersistentStorageLocationService)), Shared]
     internal class DefaultPersistentStorageLocationService : IPersistentStorageLocationService
     {
+        /// <summary>
+        /// Used to ensure that the path components we generate do not contain any characters that might be invalid in a
+        /// path.  For example, Base64 encoding will use <c>/</c> which is something that we definitely do not want
+        /// errantly added to a path.
+        /// </summary>
+        private static ImmutableArray<char> s_invalidPathChars = Path.GetInvalidPathChars().Concat('/').ToImmutableArray();
```

Comment:
```suggestion
        private static readonly ImmutableArray<char> s_invalidPathChars = Path.GetInvalidPathChars().Concat('/').ToImmutableArray();
```

Reply:
odd... i wonder why i didn't get an ide notification for that...